config/config.go

```diff
@@ -66,13 +71,19 @@ const (
 type SubscriberConfig struct {
 	Enabled  bool
 	Channels string
+	Push     bool
 }
 
 type PublisherConfig struct {
 	Enabled         bool
 	Channels        string
 	PublishInterval time.Duration
 	MessageSize     int64
+	Push            bool
+	PushAPIKey      string
+	PushURL         string
+	PushNamespace   string
+	PushDeviceID    string
```

Comment:
Should these be an object instead? For example:

```go
type PublisherConfig struct {
	// ...
	PushDevice PushDeviceConfig
}

type PushDeviceConfig struct {
	Enabled   bool
	ID        string
	APIKey    string
	URL       string
	Namespace string
}
```

Reply:
Also, I think it should be the subscriber that sets up the push devices, because in a fanout scenario, we want a single publisher but with N push subscribers, rather than vice versa.